### PR TITLE
CEDS 2295 Redesign file upload receipts page

### PR DESCRIPTION
**Pull request by glidester**
As part of public beta readiness the content of the page was requested
to be altered by the UX team.

Main changes are adding a green panel that includes the MRN and links
to the NCH and 'print this page'.

This turned out to be a larger change than first envisaged as to 
implement the green panel the new GDS frontend had to be incorporated
into the code base.

Due to time constraints upgrading all the pages to the new frontend 
was not possible.

### CODE REVIEW

**Review by PauloPortugal — CHANGES_REQUESTED** [review]
@glidester please remove the partials. I don't think we need them
:smiley:

**Review by PauloPortugal — APPROVED** [review]
@glidester please remove the `stylesheets/vendor` folder as well. I think we do not need that as well.
:wink:

File: app/views/BackButton.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import play.api.mvc.Call
+
+case class BackButton(title: String, call: Call)
```
Comment (PauloPortugal) [inline app/views/BackButton.scala:21]: Why do we need the `BackButton` class?

File: app/views/components/gds/gdsMainTemplate.scala.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    <!--[if gt IE 8]><!--><link href='@routes.Assets.versioned("stylesheets/customsdecexfrontend-app.css")' media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
+}
+
+@scripts = {
+    <script src="@routes.Assets.versioned("javascripts/show-hide-content.js")" type="text/javascript"></script>
+
+    @if(useTimeoutDialog) {
+        <script src='@routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
+        <script src='@routes.Assets.versioned("javascripts/timeoutDialog.js")'> </script>
+        <script>window.HMRCFrontend.initAll();</script>
```
Comment (PauloPortugal) [inline app/views/components/gds/gdsMainTemplate.scala.html:64]: Please remove references to `hmrc-frontend` and `initAll`, as we do not import them.

File: app/views/BackButton.scala (hunk 1 above)
Comment (glidester) [reply app/views/BackButton.scala:21]: It is used in the gdsMainTemplate. Technically it is not used atm on SFUS though so I'll remove it.

File: build.sbt [hunk 3]
```diff
@@ -38,9 +38,11 @@ lazy val microservice = (project in file("."))
 val httpComponentsVersion = "4.5.11"
 
 val compileDependencies = Seq(
-  "uk.gov.hmrc"                   %% "govuk-template"               % "5.54.0-play-26",
-  "uk.gov.hmrc"                   %% "play-ui"                      % "8.11.0-play-26",
   "uk.gov.hmrc"                   %% "bootstrap-frontend-play-26"   % "2.24.0",
+  "uk.gov.hmrc"                   %% "play-ui"                      % "8.14.0-play-26",
+  "uk.gov.hmrc"                   %% "play-frontend-govuk"          % "0.53.0-play-26",
+  "uk.gov.hmrc"                   %% "play-frontend-hmrc"           % "0.26.0-play-26",
```
Comment (PauloPortugal) [inline build.sbt:44]: We do not need this one I believe.

**Review by PauloPortugal — COMMENTED** [review]
I see we have brought quite a few classes to the SFUS. Looks good! :+1: 
Please address the comments.

File: app/views/BackButton.scala (hunk 1 above)
Comment (PauloPortugal) [reply app/views/BackButton.scala]: Great! :+1:

File: build.sbt (hunk 3 above)
Comment (glidester) [reply build.sbt:44]: You're right we don't need play-frontend-govuk, but the other two we do

File: app/views/components/gds/gdsMainTemplate.scala.html (hunk 2 above)
Comment (glidester) [reply app/views/components/gds/gdsMainTemplate.scala.html]: Ok, removed

**Review by PauloPortugal — APPROVED** [review]
LGTM :+1:

**Review by PauloPortugal — APPROVED** [review]
Well spotted! :)

**Pull request merged**